public/utils/utils.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// Append 16 random characters
+export function generateId(prefix: string) {
```

Comment:
just wondering is something like this used in other places. I know in backend we have some auto ID generator methods or using the document ID

Reply:
I haven't come across anything on OSD side but I'll check with couple of them.

Reply:
Does flowise create a node Id like this also?

Reply:
Yes- we need them to differentiate if there are multiple components of the same type in a workspace.

Reply:
There is nothing concrete on OSD side for this, and to prevent more coupling I think it is best we keep this util fn.